android/src/main/java/com/swmansion/reanimated/layoutReanimation/Snapshot.java

```diff
@@ -115,6 +124,21 @@ public Snapshot(View view) {
     }
     originXByParent = view.getLeft();
     originYByParent = view.getTop();
+    if (view.getBackground() != null) {
+      borderRadius = ((ReactViewBackgroundDrawable) view.getBackground()).getFullBorderRadius();
+    } else if (view instanceof ReactImageView) {
+      borderRadius = getImageBorderRadius((ReactImageView) view);
+    }
+  }
+
+  private float getImageBorderRadius(ReactImageView imageView) {
+    try {
+      Field field = imageView.getClass().getDeclaredField("mBorderRadius");
+      field.setAccessible(true);
```

Comment:
I remember that we needed to use `accessFlags` in similar cases for some reason, can you confirm the current code is sufficient?

Reply:
Yes, this code is sufficient. The `accessFlags` field is required when we want to get access to static field of class - https://stackoverflow.com/questions/3301635/change-private-static-final-field-using-java-reflection/64378131#64378131